### ISSUE TITLE
For the completed task to flood the of the underground Agroprom does not improve the ratio of "Debt" to the hero.

### ISSUE THREAD

**Issue by JIuBeP** [Decane/SRP#52]
For the completed task to flood the of the underground Agroprom does not improve the ratio of "Debt" to the hero (remains simply without changes).

**Comment by Marek33** [comment]
I think you mean reputation, and not the Debt, or you mean something else?

**Comment by SurDno** [comment]
Duty. It's called Duty. Debt is when you owe money to somebody.

I do not know why you keep uploading issues of faction's disposition not being raised. I don't think it's really a bug, it's more of an oversight by GSC and it's done in almost every single storyline quest.

There is no way to know if it's supposed to be this way.

**Comment by Marek33** [comment]
@SurDno  I agree with you, but I think that since you have been risking your life for them, you should get some reputation for that.

**Comment by JIuBeP** [comment]
Pardon! Yeah, you're right.

**Comment by Marek33** [comment]
@SurDno I know, but this could be in the Vanilla game, and not in a global overhaul.

**Comment by Decane** [comment]
In configs\misc\tm_agroprom_underground.ltx:
```
[agru_leader_dolg_task]
(...)
community_relation_delta_complete=+400
```
So it looks like flooding the underground _does_ increase Duty's community_goodwill towards the player.

@JIuBeP Are you sure this doesn't work? Note: the community_goodwill parameter is bounded to {-3000, 1000} in the SRP, so if you already had 1000 -- perhaps by completing side-tasks for Duty -- Duty's overall goodwill towards you wouldn't have increased any further; this is by design.

**Comment by Decane** [comment]
Correction to https://github.com/Decane/SRP/issues/52#issuecomment-515755622:

Flooding the underground increases **Clear Sky's** (not Duty's) community_goodwill towards the player, because the quest-giver for "agru_leader_dolg_task" is "csky" (instead of "dolg") in configs\scripts\agroprom_underground\agru_quest_line.ltx.
